astropy/units/function/core.py

```diff
@@ -643,7 +643,7 @@ def __le__(self, other):
         return self._comparison(other, self.value.__le__)
 
     def __lshift__(self, other):
-        """Unit converstion operator `<<`"""
+        """Unit converston operator `<<`"""
```

Comment:
Huh, that's weird. Did we break the checker?
```suggestion
        """Unit conversion operator `<<`"""
```